src/main/java/com/minecolonies/colony/ColonyManager.java

```diff
@@ -744,4 +762,25 @@ public static IMessage handleColonyViewRemoveBuildingMessage(int colonyId, Block
 
         return null;
     }
+
+    /**
+     * Returns result of {@link ColonyView#handleColonyViewRemoveWorkOrderMessage(int)} if {@link #getColonyView(int)}
+     * gives a not-null result. If {@link #getColonyView(int)} is null, returns null
+     *
+     * @param colonyId      ID of the colony
+     * @param workOrderId    ID of the workOrder
+     * @return              result of {@link ColonyView#handleColonyViewRemoveWorkOrderMessage(int)}  or null
+     */
+    public static IMessage handleColonyViewRemoveWorkOrderMessage(final int colonyId, final int workOrderId)
+    {
+        ColonyView view = getColonyView(colonyId);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'view' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)